src/main/java/org/jenkinsci/plugins/environmentdashboard/DashboardBuilder.java

```diff
@@ -260,7 +273,7 @@ private String writeToDB(Run<?, ?> build, TaskListener listener, String envName,
 			stat.execute(runQuery);
 		} catch (SQLException e) {
 			returnComment = "Error running query " + runQuery + ".";
-			return returnComment;
+			return returnComment + e;
```

Comment:
This will fail, `e` is not a string?

Reply:
Added it to print the exact exception which I was getting during debugging. Removed it now.